x-pack/plugins/cases/server/common/utils.test.ts

```diff
@@ -940,4 +941,26 @@ describe('common utils', () => {
       expect(expectedReferences).toEqual(expect.arrayContaining(updatedReferences));
     });
   });
+
+  describe('asArray', () => {
+    it('returns undefined when the field is undefined', () => {
+      expect(asArray(undefined)).toBeUndefined();
```

Comment:
I would expect the `asArray` method to return an empty array if the value is `undefined` or `null`. What do you think?